src/api/Filter.js

```diff
@@ -0,0 +1,48 @@
+import { addExtremeValuesToArr } from '../utils/addExtremeValuesToArr';
+import { FilterTypes } from './FilterQueryBuilder';
+
+const filterFunctions = {
+  [FilterTypes.IN](filterValues, featureValue) {
+    return filterValues.includes(featureValue);
+  },
+  [FilterTypes.BETWEEN](filterValues, featureValue) {
+    const checkRange = (range) => {
+      const [lowerBound, upperBound] = range;
+      return featureValue >= lowerBound && featureValue <= upperBound;
+    };
+
+    return addExtremeValuesToArr(filterValues).some(checkRange);
+  }
+};
+
+export function applyFilter({ filters = {}, returnedType = 'boolean' }) {
```

Comment:
It would be nice to reduce the size of this function, so we have smaller chunks, like for example a `passesFilter(feature)`